tests/fixtures/auth.py

```diff
@@ -35,3 +48,158 @@ def auth_credentials(fs):
     )
 
     return credentials
+
+
+@pytest.fixture
+def basic_client(monkeypatch):
+    """Select basic auth and return a TestClient."""
+    monkeypatch.delenv("RALPH_RUNSERVER_AUTH_OIDC_ISSUER_URI", raising=False)
+    monkeypatch.delenv("RALPH_RUNSERVER_AUTH_OIDC_AUDIENCE", raising=False)
+    reload(conf)
+    app = reload(api).app
+    return TestClient(app)
+
+
+@pytest.fixture
+def oidc_client(monkeypatch):
```

Comment:
I would suggest something more explicit:

```suggestion
def oidc_test_client(monkeypatch):
```